packages/contracts/scripts/util/get-deployer-wallet.ts

```diff
@@ -0,0 +1,60 @@
+require("dotenv").config();
+var readlineSync = require("readline-sync");
+import { readFileSync } from "fs";
+import { Wallet, ethers } from "ethers";
+
+// Exports a function that returns the private key of the wallet to use for
+// deploying contracts. The private key is loaded from an encrypted keystore
+// file, defined by env var `WALLET_ENCRYPTED_KEYSTORE_FILE`. If this env var
+// is not set, null is returned, and a warning will be logged.
+// @dev use closure to only ever ask for password once
+export var getDeployerWallet = ((): (() => Wallet | null) => {
```

Comment:
Nit: Any reason we're using var in this file?

Reply:
good call - fixed in [8300810](https://github.com/ArtBlocks/artblocks-contracts/pull/1206/commits/830081088301ffd0698357dead7c8c98ea9bb6ec)